src/interfaces/account.interface.mock.ts

```diff
@@ -0,0 +1,9 @@
+import { AccountTypeEnum } from '../enums/account-type.enum';
+import { AccountInterface } from './account.interface';
+
+export const mockHdAccount: AccountInterface = {
+  name: 'Mock HD Account',
+  publicKey: 'mockHdAccountPublicKey',
```

Comment:
What do you think about usage of these credentials https://github.com/madfish-solutions/templewallet-mobile/blob/5277-unit-tests-wallet-account-state/src/mocks/account-credentials.mock.ts here?

Reply:
good point